ci-usd/README.md

```diff
@@ -31,6 +31,7 @@ Contains:
 * pyside-5.12.6
 * cmake-3.12.4
 * imath-2.3.0
+* materialx-
```

Comment:
Looks like something may be broken here and it didn't pull ASWF_MATERIALX_VERSION? Or is it because this version of USD didn't require MaterialX?

Reply:
Indeed, I only added materialx version for 2022, not planning on adding it (yet) to previous versions. I need to check if the version exists in the jinja template to fix this!